lib/health_monitor/providers/database.rb

```diff
@@ -11,7 +11,11 @@ def check!
         failed_databases = []
 
         ActiveRecord::Base.connection_handler.connection_pool_list(:all).each do |cp|
-          cp.lease_connection.execute('SELECT 1')
+          if Rails::VERSION::MAJOR >= 8
```

Comment:
other option here could be `if cp.respond_to? :lease_connection`

Reply:
Perhaps this would better

Reply:
Done 👍